app/controllers/tutors_controller.rb

```diff
@@ -75,8 +75,10 @@ def add_student
 
   def tutor_params
     params.require(:tutor).permit(
-      :name,
-      :email
+      :affiliate, :first_name, :last_name, :email, :address, :city, :state,
+      :zip, :phone, :cell_phone, :gender, :native_language, :race, :referral,
+      :training_type, :education, :employment, :occupation, :orientation,
+      :training, :dob
```

Comment:
I really like making these lists one per line, alphabetized, just so that when we add and remove them in the future it makes clean diffs. But that's just a personal preference.

In vim you can select multiple lines with line selection and use `:sort` to sort lines. Dunno if you're using vim.